rasa_nlu/extractors/duckling_http_extractor.py

```diff
@@ -116,13 +116,14 @@ def _duckling_parse(self, text, reference_time):
     def _reference_time_from_message(message):
         if message.time is not None:
             try:
-                return int(message.time)
+                return int(message.time)*1000
```

Comment:
i'm not sure this makes sense? what format is the time of the message usually in?